astronomer/airflow/version_check/update_checks.py

```diff
@@ -24,19 +25,70 @@
 from airflow.utils.db import create_session
 from airflow.utils.log.logging_mixin import LoggingMixin
 from airflow.utils.timezone import utcnow
-from airflow.www import auth
+from airflow.utils.net import get_hostname
+from airflow.www.extensions.init_auth_manager import get_auth_manager
```

Comment:
This is not in older versions, therefore, it should be inside the has_access_custom.

Reply:
Thanks for pointing that out, moved it into `has_access_` method.